packages/ember-intl/tests/unit/macros/format-number-test.js

```diff
@@ -0,0 +1,54 @@
+import Ember from 'ember';
+import { moduleFor, test } from 'ember-qunit';
+import { formatNumber, raw, ref } from 'ember-intl/macros';
+
+moduleFor('service:intl', 'Unit | Macro | formatNumber', {
+  integration: true,
+  beforeEach() {
+    let intl = this.subject();
+    intl.setLocale('en');
+
+    this.object = Ember.Object.extend({
+      intl,
+      someNumber: 12345,
+      fractionDigits: 3,
+      formattedNumber: formatNumber('someNumber'),
+      rawFormattedNumber: formatNumber(raw(10001)),
```

Comment:
Is `raw` actually needed at all when there's also `ref`? It seems it would be easier to simply expect everything to be a value unless explicitly marked as a reference via `ref`?